frontend/src/core/less/fonts.less

```diff
@@ -17,27 +18,10 @@ body {
 h1 {
   font-size:15px;
   font-family: @font-title;
-  font-weight:300;
 }
 
-h2 {
-  font-weight:300;
-}
-
-h3 {
-  font-weight:300;
-}
-
-h4 {
-  font-weight:200;
-}
-
-h5 {
-  font-weight:200;
-}
-
-h6 {
-  font-weight:200;
+h1, h2, h3, h4, h5, h6 {
```

Comment:
Any reason we're changing these to use a single weight? (appreciate we weren't importing the specified weights)